chapters/ch45.md

```diff
@@ -1,38 +1,35 @@
 > # 45. General form of Optimization Verification test
 
---> _replace THIS LINE by your translation for the above line_
+ # 45. Dạng chung của thử nghiệm Xác minh tối ưu
 
 
-> You can apply the Optimization Verification test when, given some input ​*x​*, you know how to compute Score​<sub>x</sub>​(*y​*) that indicates how good a response ​*y​* is to an input ​*x*.​ Furthermore, you are using an approximate algorithm to try to find arg max​<sub>y</sub>​ Score​<sub>x</sub>​(*y​*), but suspect that the search algorithm is sometimes failing to find the maximum. In our previous speech recognition example, ​*x=A​* was an audio clip, and ​*y=S​* was the output transcript.
+> You can apply the Optimization Verification test when, given some input ​*x​*, you know how to compute Score​<sub>\*</sub>​(*y​*) that indicates how good a response ​*y​* is to an input ​*x*.​ Furthermore, you are using an approximate algorithm to try to find arg max​<sub>y</sub>​ Score​<sub>\*</sub>​(*y​*), but suspect that the search algorithm is sometimes failing to find the maximum. In our previous speech recognition example, ​*x=A​* was an audio clip, and ​*y=S​* was the output transcript.
 
---> _replace THIS LINE by your translation for the above line_
+Bạn có thể áp dụng thử nghiệm Xác minh tối ưu khi, với một số đầu vào *x*, bạn biết cách tính Điểm<sub>\*</sub> (*y*) dùng để thể hiện mức độ phản hồi *y* tốt như thế nào với *x*. Hơn nữa, bạn đang sử dụng thuật toán gần đúng để cố gắng tìm arg max <sub>y</sub>Điểm<sub>\*</sub>(*y*), nhưng nghi ngờ rằng thuật toán tìm kiếm đôi khi không tìm thấy giá trị lớn nhất. Trong ví dụ nhận dạng giọng nói trước đây của chúng tôi, *x=A* là một đoạn âm thanh và *y=S* là bản ghi đầu ra.
```

Comment:
- Gợi ý dịch gọn: 
> Bạn có thể áp dụng thử nghiệm Xác minh tối ưu khi, bạn biết cách tính Điểm<sub>\*</sub> (*y*) dùng để thể hiện mức độ phản hồi *y* tốt như thế nào với với một số đầu vào *x* .

Reply:
- Transcript dịch là bản ghi thì nghe nó có vẻ chưa mượt lắm. Ở đây giống như là input là âm thanh còn đầu ra là chữ viết vậy. 
>  Trong ví dụ nhận dạng giọng nói trước đây ~~của chúng tôi~~, *x=A* là một đoạn âm thanh và *y=S* là bản ghi đầu ra.

Reply:
@ngcthuong 
* cụm "Optimization Verification test" được dịch là "Bài kiểm tra xác minh tối ưu theo #301.

 * Từ "transcript" hiện đang được dịch là "bản ghi" hoặc "bản phiên âm", mình cần thống nhất và sửa lại chỗ này.

Reply:
Từ phiên âm là chỉ cách phát âm, nó không giống với nghĩa gốc là transcript lắm. Còn dịch transcript là bản ghi thì mình cảm giác vẫn dễ bị nhầm. Nó giống như bản chữ viết hơn.

Reply:
@ngcthuong để move forward thì mình cần chọn một cái rồi dịch lại toàn bộ. Sau nếu thấy cần thiết sẽ sửa lại sau.